quadrat/sass/block-styles/_query.scss

```diff
@@ -0,0 +1,70 @@
+.is-style-quadrat-diamond-posts {
+	.post-meta {
+		justify-content: flex-start;
+	}
+	.wp-block-query-pagination,
+	.post-meta {
+		a {
+			text-decoration: none;
+			&:hover {
+				text-decoration: underline;
+			}
+		}
+	} 
+	.wp-block-column {
+		flex-basis: auto;
+		&:last-child {
+			max-width: 400px;
+			flex-shrink: 0;
+		}
+	}
+	.wp-block-post-featured-image {
+		width: 400px;
+		@media (max-width: #{ ($break-small - 1) }) {
+			width: 100%;
+		}
```

Comment:
this aligns it to the Columns block media query that collapses the two columns into one. I tried to find a solution without a media query but I couldn't :(